.github/issue_template/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature Request
+about: Suggest an idea or enhancement for this project
+---
+
+**Is your feature request related to a problem? Please Describe.**
+
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
```

Comment:
Hey @Gautam-Arora24 ! I think you can add a screenshot row in this template as well. But, please wait because maintainers might have a better approach for this. The same goes for the `Pull Request` template (for example: there might be a test case where UI is changed and sharing a screenshot in the PR Description is always welcomed. Happy Coding!

Reply:
Sure, I am waiting for the mentors' reply. They will guide me on how to proceed further :)